int32 nSkeletons
nat_ros/skeleton[] skeletons